testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/FlowOverrideTest.java

```diff
@@ -264,6 +266,34 @@ public void testWithClientBrowserOverride() throws Exception {
         events.expectLogin().client("test-app-flow").detail(Details.USERNAME, "test-user@localhost").assertEvent();
     }
 
+    // TODO remove this once DYNAMIC_SCOPES feature is enabled by default
+    @Test
+    @EnableFeature(value = Profile.Feature.DYNAMIC_SCOPES, skipRestart = true)
+    public void testWithClientBrowserOverrideWithDynamicScope() throws Exception {
+        // Just use existing test with DYNAMIC_SCOPES feature enabled as it was failing with DYNAMIC_SCOPES
+        oauth.clientId(TEST_APP_FLOW);
```

Comment:
Thanks for this update.

One note: Is it possible to just directly call method `testWithClientBrowserOverride()` instead of copy/pasting the whole content of this method? It may be slightly better for the maintenance to avoid unecessary code duplicities.